ci/check-documentation.py

```diff
@@ -86,6 +87,31 @@ def complain(message):
         # check local link validity
 
         for url in sorted([ref.url for ref in doc_page.external_references()]):
+            if url.startswith(omz_github_url):
+                omz_relative_url = url[len(omz_github_url):]
+                omz_relative_path = Path(urllib.request.url2pathname(omz_relative_url))
+
+                if omz_relative_path.as_posix().startswith(('blob/', 'tree/')):
+                    omz_relative_path = Path(*omz_relative_path.parts[1:]) # removed blob/ or tree/
+                    omz_relative_posix_path = omz_relative_path.as_posix()
+
+                    distance_to_md_parent_dir = 0
```

Comment:
This variable is only used in one branch of the following `if`, so it should be moved there.